eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -23,14 +23,21 @@ stages:
                   artifact: ${{parameters.ArtifactName}}
                   timeoutInMinutes: 5
 
+                - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
I'd prefer we not do this here but instead do it in the build/package stage and use it to omit any packages that aren't in the ci.yml from being published to the artifacts.